docs/Config.md

```diff
@@ -53,6 +53,9 @@ oS:
   openCommand: open {{filename}}
   openLinkCommand: open {{link}}
 update:
+  refreshProjectTime: 100ms
```

Comment:
Given that a user is unlikely to know which of these if any are causing performance issues, I wonder if we should just have a single duration. This also will make the code easier to maintain, because if we add a new thing that refreshes frequently we can just use the config value.

Reply:
Jup i was thinking that too, but i was not sure if that was a good idea because the issue this solves probably only needs to change the volumes pull rate.  

But because of the *probably* in the sentence above and your point about maintainability i'll change it to a generic rule.

Reply:
sounds good :) If it turns out we want to be more granular, we can do that down the line